.github/workflows/openapi-validator.yml

```diff
@@ -1,9 +1,16 @@
 name: API validation
 
-on: [push]
+on:
+  push:
+    branches:
+      - development
+      - master
+  pull_request:
+    branches:
+      - "**"
 
```

Comment:
What is this actually doing? You did not give any rationale in the PR template for this change. So far we checked every pushed commit, does this mean we check only for Pull Requests? If so, I am not sure this is a good change. When running on push(commit) basis, you get immediate feedback and can fix things. When this check is delayed until a PR is opened, this may mean (several) force-pushed documented in the PR which is confusing.